cou/cli.py

```diff
@@ -220,8 +218,8 @@ def entrypoint() -> None:
         loop = asyncio.get_event_loop()
         loop.run_until_complete(_run_command(args))
     except HighestReleaseAchieved as exc:
+        progress_indicator.succeed()
         print(exc)
-        sys.exit(0)
```

Comment:
I think having the `exit(0)` is clearer, but I'm not against removing it. However, you would need to delete the corresponding unit test `tests/unit/test_cli.py::test_entrypoint_highest_release`

Reply:
It's default behavior, so it's not important at all.

Reply:
With your change the previous test implementation would fail. But I see you updated it, so that's good.